src/test/java/de/tum/in/www1/artemis/util/CourseTestService.java

```diff
@@ -1207,23 +1197,36 @@ public void testDownloadCourseArchiveAsInstructor_not_found() throws Exception {
 
     // Test
     public void testDownloadCourseArchiveAsInstructor() throws Exception {
+        submissionRepo.deleteAll();
```

Comment:
Apparently this causes a compilation error:
`/opt/bambooagent/bamboo-agent-home/xml-data/build-dir/ARTEMIS-TESTS1730-JAVATEST/src/test/java/de/tum/in/www1/artemis/util/CourseTestService.java:1200: error: cannot find symbol`

Reply:
Thanks, something went wrong while I resolved merge conflicts